README.md

```diff
@@ -14,6 +13,33 @@ MMEX includes all the basic features that 90% of users would want to see in a
 personal finance application. The design goals are to concentrate on
 simplicity and user-friendliness – an application that can be used everyday.
 
+
+## Features
+
+- Fast, intuitive and functional
+- Checking, credit card, savings, stock investment and asset accounts
+- Unlimited nested multi-level category support
+- Reminders for recurring bills and deposits
+- Budgeting and cash flow forecasting
+- Simple one-click reporting with graphs and charts
+- Import data from [Comma-Separated Values (CSV)](https://en.wikipedia.org/wiki/Comma-separated_values) and [Quicken Interchange Format (QIF)](https://en.wikipedia.org/wiki/Quicken_Interchange_Format) files
+- [Custom Reports](https://github.com/moneymanagerex/general-reports#general-reports)
+- Installation not required: portable versions run from a USB or a flash drive
+- Nonproprietary [SQLite](https://www.sqlite.org/) database with [Advanced Encryption Standard (AES)](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard)
+- Cross-platform, available for [desktop](https://moneymanagerex.org/docs/downloads/compatibility/) ([GNU/Linux](https://moneymanagerex.org/docs/downloads/compatibility/#linux), [macOS](https://moneymanagerex.org/docs/downloads/compatibility/#macos--osx), [Windows](https://moneymanagerex.org/docs/downloads/compatibility/#windows)) and mobile ([Android](https://android.moneymanagerex.org/)) operating systems
+- [International language support](https://crowdin.com/project/moneymanagerex)
+
+
+## Download
+
+* [Download stable release](https://github.com/moneymanagerex/moneymanagerex/releases/) for GNU/Linux, macOS and Windows
+* [Download developmental release](https://app.circleci.com/pipelines/github/moneymanagerex/moneymanagerex) for GNU/Linux
```

Comment:
Maybe this should be 
`* Download developmental release for [GNU/Linux](https://app.circleci.com/pipelines/github/moneymanagerex/moneymanagerex), [Mac](https://link.ipx.co.uk/mmex-mac-betas), or [Windows](https://ci.appveyor.com/project/whalley/moneymanagerex)`

Reply:
Thank you for your links. Should be updated.